### ISSUE TITLE
precision on kitti test set?

### ISSUE THREAD

**Issue by Z-Lee-corder** [PJLab-ADG/LoGoNet#33]
I try to train the model on KITTI with all training data.  But the precision on moderate level for car is only 82%,   I want to know how to achieve 85% as paper said?
